extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -1574,6 +1618,35 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     dev().warn(TAG, message, error);
   }
 
+  /**
+   * Generate a 32-byte random string.
+   * Uses the win.crypto when available.
+   * @return {string} The random string
+   * @private
+   */
+  getRandomString_() {
+    // 16 hex characters * 2 bytes per character = 32 bytes
+    const length = 16;
+    const randomValues = Services.cryptoFor(this.win).getSecureRandomBytes(
+      length
+    );
+
+    let randomSubdomain = '';
+    for (let i = 0; i < length; i++) {
+      // If crypto isn't available, just use Math.random.
+      const randomValue = randomValues
+        ? randomValues[i]
+        : Math.floor(Math.random() * 255);
```

Comment:
I think just using the `Math.random` is actually fine. While not actually "secure", it's still very random in browsers.

Reply:
Sounds good, I'll keep the changes that the ads team recommended